src/main/java/com/codecool/virtualpub/logic/Game.java

```diff
@@ -55,7 +55,9 @@ private List<Drink> generateDrinks(int numberOfDrinks) {
         List<Drink> stock = new ArrayList<>();
         for (int i = 0; i < numberOfDrinks; i++) {
             int randomIndex = rand.nextInt(GameData.DATA.getDrinks().size());
-            stock.add(GameData.DATA.getDrinks().get(randomIndex));
+            Drink drink = GameData.DATA.getDrinks().get(randomIndex);
+            Drink newDrink = new Drink(drink.getBrand(), drink.getDrinkType(), drink.getPrice());
```

Comment:
If you want to make a copy of the `Drink`, I advise you to make a "copy constructor" ie a constructor that takes an instance of the same class as parameter like this:
```java
public Drink(Drink other) {
    this.brand = other.brand;
    //...
}
```

And use the copy constructor here at line 59.

```java
Drink newDrink = new Drink(drink);
```